app/src/androidTest/java/com/duckduckgo/app/fire/fireproofwebsite/ui/FireproofWebsitesViewModelTest.kt

```diff
@@ -152,6 +153,18 @@ class FireproofWebsitesViewModelTest {
         assertTrue(viewStateCaptor.value.fireproofWebsitesEntities.size == 1)
     }
 
+    @Test
+    fun whenUserTapsOnFireproofSettingsSelectionThenShowFireproofOptionsDialog() {
+
+    }
```

Comment:
Should this test be implemented?

Reply:
No, it is a future test that shouldn't be there because it can't be implemented yet. I will remove it for now